ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/policy-maps-copp.j2

```diff
@@ -0,0 +1,30 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - policy map copp #}
+{% if policy_maps.copp is arista.avd.defined %}
+
+## COPP Policy Maps
```

Comment:
This is in plural but your data model suggests we can only have one? Please confirm on EOS that it is not possible to configure multiple copp policy maps. I understand that only one can be active, but it might be possible to define more than one.

Reply:
Checked on lab. We can configure only a single copp policy map. Hence changed the heading to singular.